### ISSUE TITLE
Audio driven 可以生成独立的视频吗

### ISSUE THREAD

**Issue by nicefuture2016** [Zejun-Yang/AniPortrait#48]
python -m scripts.audio2vid --config ./configs/prompts/animation_audio.yaml -W 512 -H 512 

替换自己的图片和音频后，输出结果是3个视频拼接的，我想生成图片和音频的最终视频，另外如何生成更高清的视频呢，参数是什么？感谢

**Comment by nicefuture2016** [comment]
python -m scripts.audio2vid --config ./configs/prompts/animation_audio.yaml -W 512 -H 512 -L 60

可以生成60帧的视频对吗

**Comment by Zejun-Yang** [comment]
目前audio2video方法生成30fps的视频时口型较准确。您无需调整L参数大小，生成视频后使用其他补帧方法提升到60fps即可。
另外，如果需要去掉输出结果中的pose video，对audio2video.py文件作出如下修改：
https://github.com/Zejun-Yang/AniPortrait/blob/415eb04cb246294c141c0de7032767359e64269b/scripts/audio2vid.py#L210
`video = torch.cat([ref_image_tensor, video], dim=0)`
